package.json

```diff
@@ -56,7 +68,12 @@
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-react": "^7.31.11",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "hardhat": "^2.12.6",
+    "hardhat-contract-sizer": "^2.7.0",
```

Comment:
remove 
hardhat-contract-sizer, 
hardhat-gas-reporter,
solidity-coverage,

Reply:
Getting the following error upon removing these:

Error HH801: Plugin @nomicfoundation/hardhat-toolbox requires the following dependencies to be installed: hardhat-gas-reporter, solidity-coverage.

Reply:
Keep what we need.